test/Microsoft.AspNet.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -77,6 +77,24 @@ public class BasicTests
         }
 
         [Fact]
+        public async Task ViewWithAttributePrefix_RendersWithoutIgnoringPrefix()
+        {
+            // Arrange
+            var server = TestHelper.CreateServer(_app, SiteName, _configureServices);
+            var client = server.CreateClient();
+            var expectedContent = await _resourcesAssembly.ReadResourceAsStringAsync("compiler/resources/BasicWebSite.Home.ViewWithPrefixedAttributeValue.html");
+
```

Comment:
must have been intended to hold the extra characters on the previous line :smiling_imp: